sql/database.go

```diff
@@ -7,7 +7,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/go-llsqlite/llsqlite"
+	sqlite "github.com/go-llsqlite/llsqlite"
```

Comment:
why?

Reply:
`goimports` automatically adds this when I safe the file. I assume because the package in `llsqlite` is named `sqlite` and not `llsqlite`.